src/content/1.7/code/agda/snippet24.agda

```diff
@@ -0,0 +1 @@
+fromR .fmap = _∘_
```

Comment:
This one looks different from Haskell snippet 🤔

Reply:
Yes, that's because I gave the instance a name, `fromR`, and because you can't simply write `(∘)` in Agda. (We need the underscores.)  Nonetheless, snippet24.agda does illustrate the main of snippet24.hs---that you can leave off the arguments `f` `g`.